android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -153,23 +158,29 @@ private fun SurveyQuestionsContent(questions: List<QuestionContentUiModel>) {
                 .verticalScroll(rememberScrollState())
                 .padding(horizontal = AppTheme.dimensions.mediumPadding)
         ) {
+            val uiModel = questionUiModels[index]
             Text(
-                text = questions[index].progress,
+                text = uiModel.progress,
                 color = Color.White.copy(alpha = 0.5f),
                 style = Typography.body2
             )
             Spacer(modifier = Modifier.height(8.dp))
             Text(
-                text = questions[index].title,
+                text = uiModel.title,
                 color = Color.White,
                 style = Typography.h4
             )
+            Spacer(modifier = Modifier.height(80.dp))
             Column(
-                verticalArrangement = Arrangement.SpaceAround,
-                modifier = Modifier.fillMaxSize()
+                modifier = Modifier.fillMaxSize(),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
             ) {
-                // TODO: Add answer view here!
+                Answer(type = uiModel.displayType, onAnswerClick = {
+                    onAnswerClick(Pair(index, it))
```

Comment:
Optional: another way to define a Pair

```suggestion
                    onAnswerClick(index to it)
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/116/commits/5823aa07c1c30a1dc9437a8f40ad7c587c2f3997